### DISCUSSION TITLE
How do you get the information?

### DISCUSSION THREAD

**Discussion by titof2375** [omerfaruk-aran/esphome_samsung_hvac_bus#184] [Questions] [answered]
Hello, I would like to know how to retrieve information from the Sansung air conditioning, by remote control or by the SmartThings application.

**Comment by omerfaruk-aran** [comment, marked as answer]
Hello,

It is not possible to retrieve information from the Samsung air conditioner using the remote control. However, if you make changes via the SmartThings application and have enabled logging in ESPHome, you should be able to see the changes in the log.

For more detailed information, you can check out the project here: [esphome_samsung_ac](https://github.com/lanwin/esphome_samsung_ac).

Feel free to reach out if you have more questions!

Best regards,
